zowe_conformance/participation_form.md

```diff
@@ -1,12 +1,12 @@
-# Zowe Conformance Program
+# Zowe V2 Conformance Program
```

Comment:
Let's not edit these - the program regardless of version is the Zowe Confomrance Program. The specific versions are the only things that should change.

Reply:
Ok, removed back